### PR TITLE
Fix import error

### PR DESCRIPTION
**Pull request by sachinprasadhs**
Fix import error by upgrading the version of `keras-cv` and add installation internal dependency keras-core.

As per the updated keras-cv, there is no need to provide image input shape in `StableDiffusion` for `ImageEncoder` Ref Commit # https://github.com/keras-team/keras-cv/commit/67efa6152ee0a513d79d25ef68cf2128b26a5e04
Closes: https://github.com/keras-team/keras-io/issues/1555

### CODE REVIEW

**Comment by tirthasheshpatel** [comment]
Sure @haifeng-jin! I can help :)

**Review by tirthasheshpatel — COMMENTED** [review]
Hi, @sachinprasadhs, thanks for the fix! Just a couple of comments. Otherwise, the fix looks good.

File: examples/generative/finetune_stable_diffusion.py
Comment (sachinprasadhs) [reply examples/generative/finetune_stable_diffusion.py:36]: Tensorflow 2.14 is not compatible with 0.6.4 as I have tested and mentioned the same in the today's meeting.

File: examples/generative/finetune_stable_diffusion.py
Comment (sachinprasadhs) [reply examples/generative/finetune_stable_diffusion.py:378]: I will take care of these in a new PR

Comment (sachinprasadhs) [reply examples/generative/finetune_stable_diffusion.py:378]: Thanks for bringing it up.

File: examples/generative/finetune_stable_diffusion.py
Comment (tirthasheshpatel) [reply examples/generative/finetune_stable_diffusion.py:36]: Ah, yes. Thanks for clarifying!

**Review by tirthasheshpatel — APPROVED** [review]
LGTM! Once we cut a new release for KerasCV, it would be nice to remove the 0.6.0 pins.

**Review by pcoet — APPROVED** [review]

**Comment by pcoet** [comment]
@sachinprasadhs Approved, but it looks like the docker-image tests are failing.

**Comment by tirthasheshpatel** [comment]
> Approved, but it looks like the docker-image tests are failing

Those were fixed by @sachinprasadhs in #1557. The last commit on this PR is before #1557 got merged, hence the failures.

**Review by divyashreepathihalli — APPROVED** [review]

**Comment by sachinprasadhs** [comment]
@pcoet, Can you please merge this.

**Pull request merged**
